core/src/main/scala/com/databricks/labs/remorph/generators/sql/LogicalPlanGenerator.scala

```diff
@@ -100,8 +100,8 @@ class LogicalPlanGenerator(
         }
 
         s"${leadingComment}CREATE TABLE ${ct.table_name} (${columns}${tableConstraintStrWithComma})"
-
-      case ctas: ir.CreateTableAsSelect => s"CREATE TABLE ${ctas.table_name} AS (${generate(ctx, ctas.query)})"
+      // Removed Unnecessary Braces
```

Comment:
I think we can discard this comment - a minor nit ;)

Reply:
done